### DISCUSSION TITLE
Display Image File Name from Image Slideshow Source

### DISCUSSION THREAD

**Discussion by ProgramDeez** [obsproject/obs-studio#10851] [Development Questions]
Hey there! Currently working on trying to implement this from a script perspective, as from what I could tell there is no way to do this from within the available 'Image Slideshow' Source properties and there are no 3rd party extensions/scripts that perform this functionality.

Essentially, I am trying to find a way to display the file name of the image that is currently being shown from within an image slideshow. The image slideshow is pulling from a directory rather than a static list of images entered into the 'Image Files' field, which I believe is causing some difficulties in the script that I am currently working with.

If anyone has any idea of how I can best go about achieving this, or has any insight about how to best access attributes from an Image Slideshow Source, I would greatly appreciate it! Thanks for any help in advance!

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

The #scripting channel is probably your best bet on our Discord.

Thank you!
